packages/turf-directional-mean/index.js

```diff
@@ -0,0 +1,239 @@
+"use strict";
```

Comment:
Sounds like we would now need to add `index.js` in the global `.gitignore`

Reply:
I'll merge this PR and delete it afterwards since I can't commit to @zhuang-hao-ming 's `master` branch

Reply:
Haha, I'm sorry, I forgot it. 
Adding it to the global `.gitignore` file seems to affect other modules that have not been refactored yet.

Reply:
I think right now we've got far enough with Typescript that we can't really go back, so every "patch" should also involve refactoring the Typescript before fixing the module.